CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to Thyra
+
+Thank you for considering contributing to Thyra! We welcome contributions from the community and value the time and effort you put into making this project better.
+
+## Getting Started
+
+To contribute to Thyra, you should have a basic understanding of the Go programming language and Git version control system. If you're new to Go, we recommend checking out [A Tour of Go](https://tour.golang.org/welcome/1) to get started.
+
+Before you can start contributing, you'll need to complete the following steps:
+- install the dependencies :
+```bash
+  sudo apt update
+  sudo apt install -y build-essential libgl1-mesa-dev xorg-dev p7zip
+```
```

Comment:
This is for linux only.

My suggestion:
```suggestion
- install the dependencies:
  - Ubuntu like:
```bash
  sudo apt update
  sudo apt install -y build-essential libgl1-mesa-dev xorg-dev p7zip
```